components/kubeflow-pipeline/pipeline_components/histogram-client/flw_histogram_client/client.py

```diff
@@ -0,0 +1,63 @@
+import io
+
+import numpy as np
+from flwr.client import Client
+from flwr.common import (
+    Code,
+    EvaluateIns,
+    EvaluateRes,
+    FitIns,
+    FitRes,
+    ParametersRes,
+    PropertiesIns,
+    PropertiesRes,
+    Scalar,
+    Status,
+)
+
+import data
+from fedhist import Histogram, HistogramData, HistogramGlobal
+
+
+class HistogramClient(Client):
+    def __init__(self, data_id: int) -> None:
+        self.data_id = data_id
```

Comment:
Let's remove the `data_id` parameter here, as we won't be able to determine it as part of an edge client.